pandas/core/missing.py

```diff
@@ -621,3 +622,24 @@ def fill_zeros(result, x, y, name, fill):
             result = result.reshape(shape)
 
     return result
+
+
+def maybe_fill(obj, value, errors):
```

Comment:
That would work if we had `coerce` and `raise` modes only, but with the addition of `ignore` we have to flag the non-conformal column so that the fill routine can skip it. Hence why I modified this method to return a `bool`.

How else would you implement `ignore`?